torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1140,7 +1140,8 @@ void insertDeQuantCall(Graph* graph,
     Node* dequant =
       graph->create(Symbol::aten("dequantize"), {quantized_val});
     dequant->output()->setDebugName(
-        original_val->debugName() + ".dequant." + c10::guts::to_string(i));
+        original_val->debugName() + ".dequant." + c10::guts::to_string(i))
```

Comment:
If this was an issue that needed a fix, suggest having a simple test that checks for the type

Reply:
this actually only can happen when we do swap dequantize for list of tensors, and it will be tested later in the stack